client/src/main/java/org/asynchttpclient/ClientStats.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010 Ning, Inc.
```

Comment:
Nope. Ning hasn't been involved in here for years. I've never even been in touch with them.
Code that goes back to their sponsorship still have their copyright, but all new code goes under `AsyncHttpClient Project`